.github/workflows/goreleaser.yml

```
@@ -10,15 +10,11 @@ jobs:
   goreleaser:
     runs-on: ubuntu-latest    
     steps:
-      - name: Set up Go
-        uses: actions/setup-go@v2
-        with:
-          go-version: 1.15.2
       - uses: actions/checkout@v2
-      - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v2
-        with:
-          version: latest
-          args: release --rm-dist
-        env:
-          GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
+      - name: release dry run
+          run: make release-dry-run
```

```suggestion
        run: make release-dry-run
```
I got
```
Invalid workflow file
You have an error in your yaml syntax on line 15
```
should be related to indentation 
same for below